polyply/src/gen_coords.py

```diff
@@ -38,15 +37,6 @@ def gen_coords(args):
                    'connected by bonds, constraints or virual-sites')
             raise IOError(msg.format(molecule.name))
 
-    # renumber the resiudes so that all molecules
-    # start with residue index 1
-    for meta_molecule in topology.molecules:
-        molecule = meta_molecule.molecule
-        resids = nx.get_node_attributes(molecule, "resid")
-        offset = min(resids.values()) - 1
-        new_resids = {node: resid - offset for node, resid in resids.items()}
-        nx.set_node_attributes(molecule, new_resids, "resid")
```

Comment:
I assume this was moved?

Reply:
Actually this is not relevant anymore because the resid gets pulled from the residue graph node attribute not making the assumption that the node-key is also the resid. Thus if your structure starts not with resid 1 that is completely fine now.